include/swift/Parse/Parser.h

```diff
@@ -961,7 +961,6 @@ class Parser {
   bool parseDifferentiationParametersClause(
       SmallVectorImpl<ParsedAutoDiffParameter> &params, StringRef attrName);
 
-  /// SWIFT_ENABLE_TENSORFLOW
```

Comment:
On line 949, there is a `SWIFT_ENABLE_TENSORFLOW` flag already there and this seems to already be part of the same code block